### ISSUE TITLE
POI-Listen

### ISSUE THREAD

**Issue by kjoscha** [kjoscha/lichtenberg#25]
Ich habe einfach mal ein neues Feature eingebaut. Da es ja nicht immer auf die räumliche Position von Orten ankommt, ist es wichtig, POI's auch einfach als Liste bekommen zu können. Der Ansatz davon ist schon Live: Wenn auf eine Kategorie geklickt wird, werden die features nun rechts aufgelistet und nach Klick in die Mitte der Karte geschwenkt...

Des weiteren wäre aber eine Liste mit Beschreibungen etc. zu Exportzwecken cool...

**Comment by andneuma** [comment]
Jo, das is cool. Im Prinzip könnte sich dann auch gleich das Description-Fenster öffnen

Problem: Für Lichtenberg ist das vielleicht noch ok, wenn man das aber allein auf 100 Einträge in einer Stadt wie Berlin skaliert wirds schnell unübersichtlich. Wenn das dann noch auf ganz Deutschland übertragen wird ists ätzend.

Idee: Man zeigt die n nächsten POIs der letzten Besuchskoordinate an?

**Comment by kjoscha** [comment]
lieber der aktuellen Kartenposition, oder?

**Comment by kjoscha** [comment]
> Jo, das is cool. Im Prinzip könnte sich dann auch gleich das Description-Fenster öffnen

Done (evtl. Cache löschen, damit das funktioniert)
